package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint-plugin-import",
-  "version": "2.14.0",
+  "version": "2.14.1",
```

Comment:
Please revert this; package version numbers should never be updated in PRs.

Reply:
Reverted